java/parse-interface/src/main/java/org/sqlflow/parser/parse/InputOutputTables.java

```diff
@@ -0,0 +1,8 @@
+package org.sqlflow.parser.parse;
+
+import java.util.ArrayList;
+
+public class InputOutputTables {
+  public ArrayList<String> inputTables;
```

Comment:
ArrayList -> List ?

Reply:
Thanks, I also checked out this post: https://stackoverflow.com/questions/2279030/type-list-vs-type-arraylist-in-java